zipline-loader/src/jniMain/kotlin/app/cash/zipline/loader/internal/EcdsaP256.kt

```diff
@@ -68,7 +69,12 @@ class EcdsaP256(
   }
 
   override fun verify(message: ByteString, signature: ByteString, publicKey: ByteString): Boolean {
-    return verify(publicKey.decodeAnsiX963(), signature, message)
+    val ecPublicKey = publicKey.decodeAnsiX963()
+    return try {
+      verify(ecPublicKey, signature, message)
+    } catch (e: SignatureException) {
+      false // Malformed signature.
```

Comment:
Yay, Wycheproof caught a place where we crashed instead of returning false!

Reply:
Nice